.github/workflows/main.yml

```diff
@@ -222,21 +214,35 @@ jobs:
     name: Release
     runs-on: ubuntu-latest
     needs:
-      - build-and-test
+      - build-binaries
+      - build-and-publish-npm-packages
     if: startsWith(github.ref, 'refs/tags/v')
     steps:
+      - uses: actions/checkout@v2
+
+      - name: Plugin list
+        id: plugin-list
+        run: echo "::set-output name=list::$(find ./ -type d -name "taqueria-plugin-*" -not -path "*-util")"
+
+      - name: Check tag suffix
+        id: check-tag-suffix
+        # Below we check if the tag ends with a version number.
+        # If there is nothing after the version number the 'is-latest' output is set to true
+        run: |
+          if [[ $GITHUB_REF =~ v[0-9]\.[0-9]\.[0-9]- ]] ; then echo "::set-output name=npm-tag::next"; fi
+
       - name: Download binaries 
         id: download-binaries 
         run: |
           mkdir release && cd release
-          wget ${{ needs.build-and-test.outputs.public-url-ubuntu }} -O taqueria.${GITHUB_REF/refs\/tags\//}-linux
-          wget ${{ needs.build-and-test.outputs.public-url-windows }} -O taqueria.${GITHUB_REF/refs\/tags\//}-windows.exe
-          wget ${{ needs.build-and-test.outputs.public-url-macos }} -O taqueria.${GITHUB_REF/refs\/tags\//}-macos
+          wget ${{ needs.build-binaries.outputs.public-url-ubuntu }} -O taq.${GITHUB_REF/refs\/tags\//}-linux
+        # wget ${{ needs.build-binaries.outputs.public-url-windows }} -O taq.${GITHUB_REF/refs\/tags\//}-windows.exe
+        # wget ${{ needs.build-binaries.outputs.public-url-macos }} -O taq.${GITHUB_REF/refs\/tags\//}-macos
```

Comment:
You can re-include the Windows and MacOS steps now that the repo is public

Reply:
These have been added once again